mne/tests/test_evoked.py

```diff
@@ -399,31 +388,37 @@ def test_equalize_channels():
 
 
 def test_evoked_arithmetic():
-    """Test evoked arithmetic
-    """
+    """Test evoked arithmetic"""
     ev = read_evokeds(fname, condition=0)
     ev1 = EvokedArray(np.ones_like(ev.data), ev.info, ev.times[0], nave=20)
     ev2 = EvokedArray(-np.ones_like(ev.data), ev.info, ev.times[0], nave=10)
 
     # combine_evoked([ev1, ev2]) should be the same as ev1 + ev2:
     # data should be added according to their `nave` weights
     # nave = ev1.nave + ev2.nave
-    ev = ev1 + ev2
+    with warnings.catch_warnings(record=True):  # deprceation no weights
+        ev = combine_evoked([ev1, ev2])
     assert_equal(ev.nave, ev1.nave + ev2.nave)
     assert_allclose(ev.data, 1. / 3. * np.ones_like(ev.data))
-    ev = ev1 - ev2
-    assert_equal(ev.nave, ev1.nave + ev2.nave)
-    assert_equal(ev.comment, ev1.comment + ' - ' + ev2.comment)
-    assert_allclose(ev.data, np.ones_like(ev1.data))
+
+    # with same trial counts, a bunch of things should be equivalent
+    for weights in ('nave', 'equal', [0.5, 0.5]):
+        ev = combine_evoked([ev1, ev1], weights=weights)
+        assert_allclose(ev.data, ev1.data)
+        assert_equal(ev.nave, 2 * ev1.nave)
+        ev = combine_evoked([ev1, -ev1], weights=weights)
+        assert_allclose(ev.data, 0., atol=1e-20)
+        assert_equal(ev.nave, 2 * ev1.nave)
+    ev = combine_evoked([ev1, ev1], weights=[0.5, -0.5])
+    assert_allclose(ev.data, 0., atol=1e-20)
+    assert_equal(ev.nave, 2 * ev1.nave)
 
     # default comment behavior if evoked.comment is None
     old_comment1 = ev1.comment
     old_comment2 = ev2.comment
     ev1.comment = None
-    with warnings.catch_warnings(record=True):
-        warnings.simplefilter('always')
-        ev = ev1 - ev2
-        assert_equal(ev.comment, 'unknown')
+    ev = combine_evoked([ev1, ev2], weights=[1, -1])
+    assert_true(ev.comment is None)
```

Comment:
I thought about this... we could do something like this for arbitrary weights:
```
evoked.comment = ' + '.join('%0.3f * %s' % (w, ev.comment) for w, ev in zip(...)
```
It would be a bit verbose, but I can't think of anything better.

If it's equal weighting, we could use that, or leave off the `'%0.3f'`.

For nave-weighting, I'm not sure the best thing.